src/app/Main/Body/InfoPage/InfoPage.model.tsx

```diff
@@ -15,16 +15,26 @@ export interface AmrsTable {
     titleString: string;
     description: string;
     tableHeader: string[];
+    tableSubHeader: string[];
     tableRows: AmrsTableData[];
-    commentText: string;
 }
 
-export type AmrKey = "coliSalm" | "coliSalmTwo" | "campy" | "mrsa" | "ef";
+export type AmrKey =
+    | "table1Coli"
+    | "table2Salm"
+    | "table3aCampy"
+    | "table3bCampy"
+    | "table4Mrsa"
+    | "table5Ef";
 
 export const microorganismNames: Record<string, JSX.Element> = {
     Salm: <i>Salmonella</i>,
     Campy: <i>Campylobacter</i>,
-    Listeria: <i>Listeria monocytogenes</i>,
+    Listeria: (
+        <i>
+            Listeria (<i>L.</i>)monocytogenes
```

Comment:
Reject: italic in italic? Did you meant `<i>Listeria (</i>L.<i>)monocytogenes</i>`

Reply:
Done